README.md

```diff
@@ -63,6 +63,10 @@ Certificate password. Default `""`.
 
 Output path of ipa. Default `"output.ipa"`.
 
+### `disable-targets`
+
+These targets will not use automatic code signing and instead use the identity specified in other inputs. Input targets separated by ','. For example, 'MyApp,YourApp'. Default: ""
```

Comment:
I think it would be easier to understand if write that all targets are disabled auto signing by default.

```suggestion
These targets will not use automatic code signing and instead use the identity specified in other inputs. Input targets separated by ','. For example, 'MyApp,YourApp'. Default "".  (default to all targets)
```